app/templates/login.hbs

```diff
@@ -94,28 +94,30 @@
     onClose=closeModal
     translucentOverlay=true
   }}
-    <h2>Forgot password</h2>
+    <h2 class="modal-header">Forgot password</h2>
     {{#let forgotPassword as |task|}}
       <form onsubmit={{prevent-default (perform task)}}>
         <legend>
-          First we need the e-mail address on your account
+          Enter your email address below and we'll email the instruction for setting a new one.
         </legend>
         <div>
-          <label>email address:</label>
+          {{!-- <label>Email address:</label> --}}
           <input
             type="email"
             required
-            oninput={{action (mut modalContext.email) value="target.value"}}>
+            oninput={{action (mut modalContext.email)
+            value="target.value"}}
+            placeholder="youremailaddress@gmail.com">
         </div>
-        <input
-          type="button"
-          value="Cancel"
-          onclick={{action closeModal}}
-          disabled={{task.isRunning}}>
-        <input
-          type="submit"
-          value={{if task.isRunning "Submitting..." "Submit"}}
-          disabled={{task.isRunning}}>
+          <input
```

Comment:
The indentation here is wrong (these two inputs are indented by two extra spaces). Once I get the tests all set up, they should flag incorrect indentation in the templates, but for now I'll just comment on them in pull requests 😄

Reply:
Fixed the indentation.